plugins/generic/pln/PLNGatewayPlugin.inc.php

```diff
@@ -111,28 +112,52 @@ function getManagementVerbs() {
 	 * Handle fetch requests for this plugin.
 	 */
 	function fetch() {
-                $templateMgr =& TemplateManager::getManager();
+		$plugin = & $this->getPLNPlugin();
+		$templateMgr = & TemplateManager::getManager();
+
+		$journal = & Request::getJournal();
+		$templateMgr->assign_by_ref('journal', $journal);
+
+		$pluginVersionFile = $this->getPluginPath() . DIRECTORY_SEPARATOR . '/version.xml';
```

Comment:
`DIRECTORY_SEPARATOR` will be `/` anyway -- looks like you'll have two in a row.